### PR TITLE
Federation errata

### PR DESCRIPTION
**Pull request by jricher**

### CODE REVIEW

File: sp800-63c/sec1_2_introduction.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ This document, SP 800-63C, provides requirements to identity providers (IdPs) an
 
 *This section is informative.*
 
-Federation is a process that allows for the conveyance of authentication and subscriber attribute information across networked systems. In a federation scenario, the verifier or CSP is referred to as an identity provider, or IdP. The RP is the party that receives and uses the information provided by the IdP.
+Federation is a process that allows for the conveyance of authentication attributes and subscriber attributes across networked systems. In a federation scenario, the verifier or CSP is referred to as an identity provider, or IdP. The RP is the party that receives and uses the information provided by the IdP.
```
Comment (jimfenton) [inline sp800-63c/sec1_2_introduction.md:19]: Do you want to make a similar change in the abstract and introduction sections as well as Section 2?

File: sp800-63c/sec4_fal.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ IdPs MAY establish whitelists of RPs authorized to receive authentication and at
...
 
-A subscriber's information SHALL NOT be transmitted between IdP and RP for any purpose other than those described in [Section 5.2](#privacy-reqs), even when those parties are whitelisted.
+A subscriber's information SHALL be transmitted between IdP and RP only for an identity federation transactions or functions such as identification of compromised accounts as discussed in [Section 5.2](#privacy-reqs). A subscriber's information SHALL NOT be transmitted for any other purposes, even when those parties are whitelisted.
```
Comment (jimfenton) [inline sp800-63c/sec4_fal.md]: "an identity federation transactions": remove the "an".

File: sp800-63-3/definitions.md [hunk 3]
```diff
@@ -99,6 +99,9 @@ The property that data originated from its purported source.
 #### Authoritative Source
 An entity that has access to, or verified copies of, accurate information from an issuing source such that a CSP can confirm the validity of the identity evidence supplied by an applicant during identity proofing. An issuing source may also be an authoritative source. Often, authoritative sources are determined by a policy decision of the agency or CSP before they can be used in the identity proofing validation phase.
 
+#### Authorization Credential
+A credential issued to an RP by an IdP during an identity federation transaction that grants the RP authorized access to a set of APIs (e.g., an OAuth access token). This credential can be separate from the assertion provided by the federation protocol (e.g., an OpenID Connect ID Token).
```
Comment (jimfenton) [inline sp800-63-3/definitions.md]: "A credential issued" sounds very -63A-like. I'm concerned about overuse of the word "credential". We probably need to discuss this one.

**Review by jimfenton — CHANGES_REQUESTED** [review]
Comments on some of the commits included. Also, I'd like to include an errata table (see page vii of the PDF version) this time around to be more consistent between the PDF and online. Please include the things in the table from the first errata round as well.

File: sp800-63-3/definitions.md (hunk 3 above)
Comment (jricher) [reply sp800-63-3/definitions.md]: I never liked "authorization credential" and I struggled with the right term for this piece. We settled on this and I'm still not sure, but I thought it made sense for the definition to use the same term. What we're really talking about is an OAuth token, but can we just call that out, or call it an "authorization token"?

Comment (jricher) [reply sp800-63-3/definitions.md]: Changed to "Authorization component"

File: sp800-63-3/errata.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+<div class="text-center" markdown="1">
+## Errata
+</div> 
+
+This table contains changes that have been incorporated into Special Publication 800-63-3. Errata updates can include corrections, clarifications, or other minor changes in the publication that are either editorial or substantive in nature.
+
+|Date|Type|Change|Location
+|----|----|----|----|
+|2017-12-01|Editorial|Removed the term ‘cryptographic’ from the AAL3 description.|Executive Summary|
```
Comment (jimfenton) [inline sp800-63-3/errata.md]: Please change to "straight" single quotes

File: sp800-63-3/errata.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+||Editorial|Updated reference to Risk Management Framework|§5|
+||Editorial|Fixed verbiage in xAL flowcharts|Figures6-1, 6-2, and 6-3|
```
Comment (jimfenton) [inline sp800-63-3/errata.md]: Space after "Figures"

File: sp800-63-3/errata.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+||Editorial|Added NISTIR 8062 as a reference|§8.1|
+||Editorial|Added definitions for disassociability, manageability, processing, and predictability|Appendix A|
+|2019-XX-XX|Added definition for authorization component|Appendix A|
```
Comment (jimfenton) [inline sp800-63-3/errata.md]: Perhaps add change of FAL3 wording (line 64 below)

File: sp800-63a/errata.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+<div class="text-center" markdown="1">
+## Errata
+</div> 
+
+This table contains changes that have been incorporated into Special Publication 800-63C. Errata updates can include corrections, clarifications, or other minor changes in the publication that are either editorial or substantive in nature.
+
+|Date|Type|Change|Location
+|----|----|----|----|
+|2017-12-01|Editorial|Made minor grammatical edits throughout the document.|N/A|
+||Editorial|Changed ‘Normative’ to ‘Informative’|Table 2-1|
```
Comment (jimfenton) [inline sp800-63a/errata.md]: Please use straight single quotes (here and in following lines)

File: sp800-63a/errata.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+||Substantive|Changed ‘Normative’ to ‘Informative’|§4.1|
+||Editorial|Confirmed‘Normative’|§4.2|
```
Comment (jimfenton) [inline sp800-63a/errata.md]: Space after Confirmed

File: sp800-63a/errata.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+||Substantive|Clarifiedthe requirements about processing of attributes|§4.2 Bullet 4
+||Substantive|Reduced rigor in requirement|§4.3|
+||Substantive|Clarified and removed ambiguity in requirement|§4.4|
+||Substantive|Clarified requirement|§4.4.1.3|
+||Substantive|Clarified and removed ambiguity in requirement|§4.4.1.6|
+||Substantive|Updated the section to be ‘Informative’|§6|
+||Substantive|Changed the title to processing limitation; clarified the language, incorporated privacy objectives language, and specified that consent is explicit|§8.3|
+||EditorialAdded NISTIR 8062 as a reference|§10.1|
```
Comment (jimfenton) [inline sp800-63a/errata.md]: Vertical bar after Editorial

File: sp800-63b/errata.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+<div class="text-center" markdown="1">
+## Errata
+</div> 
+
+This table contains changes that have been incorporated into Special Publication 800-63C. Errata updates can include corrections, clarifications, or other minor changes in the publication that are either editorial or substantive in nature.
+
+|Date|Type|Change|Location
+|----|----|----|----|
+|2017-12-01|Editorial|Updated AAL descriptions for consistency with other text in document|Introduction|
+||Editorial|Deleted “cryptographic” to consistently reflect authenticator options at AAL3|§4.3|
```
Comment (jimfenton) [inline sp800-63b/errata.md]: Please use straight quotes, here and below

File: sp800-63c/sec6_assertions.md [hunk 11]
```diff
@@ -5,7 +5,7 @@
...
+An assertion used for authentication is a packaged set of attribute values or attribute references about or associated with an authenticated subscriber that is passed from the IdP to the RP in a federated identity system. Assertions contain a variety of information, including: assertion metadata, attribute values and attribute references about the subscriber, information about the subscriber's authentication at the IdP, and other information that the RP can leverage (such as restrictions and expiration time). While the assertion's primary function is to authenticate the user to an RP, the information conveyed in the assertion can be used by the RP for a number of use cases &mdash; for example, authorization or personalization of a website. These guidelines do not restrict RP use cases nor the type of protocol or data payload used to federate an identity, provided the chosen solution meets all mandatory requirements contained herein.
```
Comment (jimfenton) [inline sp800-63c/sec6_assertions.md:8]: Should this change be called out in the errata table?

File: sp800-63c/sec9_privacy.md [hunk 12]
```diff
@@ -12,7 +12,7 @@ Federation offers numerous benefits to RPs and subscribers, but requires subscri
...
 
-Consult your SAOP if there are questions about whether the proposed processing falls outside the scope of the permitted processing or the appropriate privacy risk mitigation measures.
+Consult the SAOP if there are questions about whether the proposed processing falls outside the scope of the permitted processing or the appropriate privacy risk mitigation measures.
```
Comment (jimfenton) [inline sp800-63c/sec9_privacy.md:15]: Maybe say something in errata table about removal of personal pronouns?

**Review by jimfenton — CHANGES_REQUESTED** [review]
Thanks for adding the errata tables. Usual nitpicky comments attached.

File: sp800-63-3/errata.md (hunk 4 above)
Comment (jricher) [reply sp800-63-3/errata.md]: Copy-paste error from the PDF, thanks.

**Comment by jimfenton** [comment]
There were a few smart/curly quotes still there, some of which were pre-existing. I think my script has caught all of them (for now).

**Review by jimfenton — APPROVED** [review]
All set now.

**Pull request merged**